doc/userguide/output/eve/eve-json-output.rst

```diff
@@ -264,6 +264,19 @@ enabled, then the log gets more verbose.
 
 By using ``custom`` it is possible to select which TLS fields to log.
 
+Drops
+~~~~~
+
+Drops are event types logged when the engine drops a packet.
```

Comment:
Is it like `event_type == drop`? or an alert indicating that the packet was dropped?

Reply:
Sorry for not seeing this question before >___<'
This is the for `event_type == drop`, which Suri issues when a packet is dropped (caused by a rule or another reason).